examples/layout_table_example.py

```diff
@@ -7,7 +7,7 @@ class TableWindow(Gtk.Window):
     def __init__(self):
         Gtk.Window.__init__(self, title="Table Example")
 
-        table = Gtk.Table(3, 3, True)
+        table = Gtk.Table.new(3, 3, True)
```

Comment:
Use keyword arguments instead of `.new`.

Reply:
If `.new` is not used then we miss the `DeprecationWarning ` for `GtkTable`

```
python3 examples/layout_table_example.py 
examples/layout_table_example.py:11: DeprecationWarning: Gtk.Table.new is deprecated
  table = Gtk.Table.new(3, 3, True)
```
Also need to use the non-obvious `n_` prefix for attributes:
```
examples/layout_table_example.py:11: PyGTKDeprecationWarning: The keyword(s) "columns, rows" have been deprecated in favor of "n_columns, n_rows" respectively. See: https://wiki.gnome.org/PyGObject/InitializerDeprecations
  table = Gtk.Table(rows=3, columns=3, homogeneous=True)
```